app/src/main/res/layout-sw600dp-land/topic_revision_summary_view.xml

```diff
@@ -29,14 +29,15 @@
       android:layout_width="match_parent"
       android:layout_height="wrap_content">
 
-      <ImageView
+      <org.oppia.app.customview.LessonThumbnailImageView
```

Comment:
we can do the reformat in this file, else looks good.

Reply:
Reformatted all 4 files of `topic_revision_summary_view.xml`